bokehjs/src/lib/models/annotations/color_bar.ts

```diff
@@ -202,7 +202,7 @@ export class ColorBarView extends BaseColorBarView {
     }
 
     this._set_canvas_image()
-    this.plot_view.request_layout() // this.request_render()
+    this.plot_view.request_layout() // this.request_paint()
```

Comment:
We used to have `render()` method that created DOM elements (in `UIElement`, `DOMNode` model hierarchies) and for painting to canvas (`Renderer` model hierarchy). Now that renderers do both, then we have a conflict. The less invasive change was to rename `render()` in renderers to `paint()`. I've been using "paint" related terminology for awhile now in relation to canvas. Thus:
* `render()` -> create DOM elements
* `paint()` -> manipulate canvas' context

This naming convention is consistent with other libraries and frameworks, but it makes `Renderer` and related models and properties (`Plot.renderers`) a little confusing. I don't expect us to rename everything to e.g. `Painter`, `Plot.painters`, though it would be a logical choice in an ideal world. I'm open to suggestions how to move forward with this.

Reply:
"Renderer" was already borrowed from other vis tools in 2012, it's pretty common in that space so I agree that renaming user-facing types is probably not advised. A delineation between`render()` and `paint()` seems very helpful but are not really user-visible, so I don't think any tension with the user-facing objects is a concern beyond being a little annoying to us.